.github/workflows/lint.yml

```diff
@@ -62,7 +62,8 @@ jobs:
           -p cl.audio \
           -p cl.opinion_page \
           -p cl.simple_pages \
-          -p cl.recap_rss
+          -p cl.recap_rss \
+          -p cl.citations
```

Comment:
Nice, this is great, but could you alphabetize it?

Reply:
Ah yes.